src/crawlee/parsel_crawler/__init__.py

```diff
@@ -1,10 +1,11 @@
 try:
     from ._parsel_crawler import ParselCrawler
     from ._parsel_crawling_context import ParselCrawlingContext
+    from ._parsel_parser import ParselParser
 except ImportError as exc:
     raise ImportError(
         "To import anything from this subpackage, you need to install the 'parsel' extra."
         "For example, if you use pip, run `pip install 'crawlee[parsel]'`.",
     ) from exc
 
-__all__ = ['ParselCrawler', 'ParselCrawlingContext']
+__all__ = ['ParselCrawler', 'ParselCrawlingContext', 'ParselParser']
```

Comment:
Do we really want to export the parser? I'm not sure anyone wants to use it directly.

Reply:
Makes sense. Done